NuGet.config

```diff
@@ -13,8 +13,11 @@
     <add key="skiasharp" value="https://pkgs.dev.azure.com/xamarin/public/_packaging/SkiaSharp/nuget/v3/index.json" />
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="dotnet-libraries-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries-transport/nuget/v3/index.json" />
-    <!-- Added manually for .NET 8 MAUI -->- 
-    <add key="darc-pub-dotnet-maui-a33a875e" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-maui-a33a875e/nuget/v3/index.json" /></packageSources>
+    <!-- Added manually for .NET 8 MAUI --> 
+    <add key="darc-pub-dotnet-maui-a33a875e" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-maui-a33a875e/nuget/v3/index.json" />
+    <!-- Added manually for .NET 8.0.12 -->
+    <add key="darc-pub-dotnet-runtime-c1ae9626" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-c1ae9626/nuget/v3/index.json" />
```

Comment:
What needed this feed? The `build.cmd` below?

Does it use the system-installed .NET on the CI machine or something else?

Reply:
the build.cmd by default will use the existing dotnet if it exists but I don t really want that for now. So I bumped do t make it fix our build here:

https://dev.azure.com/dnceng/internal/_build/results?buildId=2599947&view=results

Reply:
Fixed to 101